src/odb/src/codeGenerator/schema/chip/dbModBterm.json

```diff
@@ -0,0 +1,40 @@
+{
+	  "name":"dbModBTerm",
+          "type":"dbObject",
+	  "fields":[
+           {
+          "name":"_name",
+          "type":"char *",
+          "flags":["no-set"]
+           },
+	      {
+		  "name":"_direction",
+		  "type":"char *"
+	      },
+
+           {
+	       "name":"_parent",
+	       "type":"dbId<_dbModule>",
+	       "flags":["no-set"],
+	       "parent":"dbBlock"
+           },
+           {
+	       "name":"_net",
+	       "type":"dbId<_dbModNet>",
+	       "parent":"dbBlock"
+           },
+	      
+	      {
+		  "name":"_next_entry",
+		  "type":"dbId<_dbModBTerm>",
+		  "flags":["no-set","no-get"]
+              } ,
+	      {
+          "name":"_modbterm_next",
```

Comment:
What is the difference between _modbterm_next and _next_entry ?

Reply:
Removed redundant modbterm list entry